src/client/java/daos-java/src/main/native/io_daos_dfs_DaosFsClient.c

```diff
@@ -1064,7 +1064,7 @@ Java_io_daos_dfs_DaosFsClient_dfsReadDir(JNIEnv *env, jobject client,
 					break;
 				}
 			}
-			if (buffer[0]) strcat(buffer, ",");
+			if (buffer[0]) strcat(buffer, "//");
```

Comment:
(style)  trailing statements should be on next line

Reply:
false alarm